src/js/tech/tech.js

```diff
@@ -313,6 +345,30 @@ class Tech extends Component {
     }));
   }
 
+
+  /**
+   * Initialize audiotrack listeners
+   *
+   * @method initAudioTrackListeners
+   */
+  initAudioTrackListeners() {
```

Comment:
These initializers are very similar and not 100% trivial. Is it possible for them to share more code?